### PR TITLE
feat: add use_devtag()

### LINKED ISSUE #2: Implement use_devtag()
So we don't have to edit DESCRIPTION and .Rproj

### PR DESCRIPTION
**Pull request by maelle**
Fix #2 

I have removed the Rproj editing because I've found that it's useless these days, `devtools::document()` and the RStudio shortcuts actually do the right things.

### CODE REVIEW

File: R/use-devtag.R [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+#' This functions amends DESCRIPTION to set up devtag usage.
+#' Call it once when you're setting a new package.
+#'
+#' @param path character path to the package
+#'
+#' @return Nothing: called for file system side effects.
+#' @export
+#'
+use_devtag <- function(path = ".") {
+  rlang::check_installed("desc")
```
Comment (maelle) [inline R/use-devtag.R:12]: or make it an Imports?

**Comment by maelle** [comment]
One thing I don't get is why in igraph I added "packages" which apparently is useless?!

**Comment by moodymudskipper** [comment]
Thanks so much!

I don't really want to test it so I'll just trust you :D. 

My few comments are : 
* can we maybe just advertise the use_devtag() way, I don't see the value of the old way
* Suggests and the CRAN transition
* Add yourself to contributors :D

**Comment by maelle** [comment]
> Suggests and the CRAN transition

What do you mean?

I have tested it but I think I'll add snapshot tests just to be on the safe side. :grin:

**Comment by moodymudskipper** [comment]
I mean that we could just add devtag to Suggests of the target package,
rather than the dependency on GitHub you're using now. It will work as soon
as the package hits CRAN

**Comment by moodymudskipper** [comment]
But maybe I misunderstood something, I saw now your comment in the ticket

**Comment by maelle** [comment]
I still think it should be a special dependency, not a Suggests, as it's only needed for building the package.

**Comment by maelle** [comment]
@moodymudskipper I added tests! It'd be great if you tested `document()` works for you in the absence of Rproj edit in a project of yours just to be sure.

File: DESCRIPTION [hunk 2]
```diff
@@ -1,13 +1,23 @@
 Package: devtag
 Title: Restrict Help Files to Development
 Version: 0.0.0.9000
-Authors@R: 
-    person("Antoine", "Fabri", , "antoine.fabri@example.com", role = c("aut", "cre"))
+Authors@R: c(
+    person("Antoine", "Fabri", , "antoine.fabri@example.com", role = c("aut", "cre")),
```
Comment (maelle) [inline DESCRIPTION:5]: that email address looks suspicious :joy:

File: DESCRIPTION [hunk 3]
```diff
@@ -1,13 +1,23 @@
...
 Encoding: UTF-8
-RoxygenNote: 7.2.3
+RoxygenNote: 7.3.2
 Imports: 
     roxygen2
 URL: https://github.com/moodymudskipper/devtag
 BugReports: https://github.com/moodymudskipper/devtag/issues
+Roxygen: list(markdown = TRUE, roclets = c("collate", "rd", "namespace", "devtag::dev_roclet"))
+Suggests: 
+    desc,
```
Comment (maelle) [inline DESCRIPTION:19]: should we just import desc

**Comment by maelle** [comment]
Friendly reminder @moodymudskipper :smile_cat:

**Comment by moodymudskipper** [comment]
Thanks!

**Pull request merged**